src/blocks/custom/checkbox/checkbox.php

```diff
@@ -39,9 +40,11 @@ class="<?php echo esc_attr( "{$classes} {$block_class}__checkbox js-{$block_clas
       <?php echo esc_attr( $is_disabled ); ?>
```

Comment:
You could however, use the functions from Denis' comment here, where you output the attributes

Reply:
Yeah, that was my intention, but I don't think that we gain much (other than following the 'WordPress way').

Reply:
See my comment from https://github.com/infinum/eightshift-forms/pull/9#discussion_r485419154

I mean the equivalent would be (which is a bit ugly IMO):

```php
$is_checked = checked( $attributes['isChecked'] ?? false, true, false );

// Render / markup starts

<?php echo esc_attr( $is_checked ); ?>
```

If we did just use `checked` on output it would be cleaner BUT then we're dealing with attributes inside the view which I'm not a big fan off (and no other block of ours does that as far as I know):

```php

// Render / markup starts

<?php checked( $attributes['isChecked'] ?? false ); ?>